classes/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
```

Comment:
`:rentals` should be added to the `attr_accessor` here to allow access to viewing and adding rentals 👍🏽